x-pack/plugins/ml/public/datavisualizer/datavisualizer.html

```diff
@@ -29,63 +29,107 @@ <h1>{{indexPattern.title}}</h1>
+              defaultMessage: 'Selecting a smaller sample size will reduce query run times and the load on the cluster.'
+            } }}"
+          ></i>
         </div>
 
       </div>
 
       <div class="kuiPanel kuiVerticalRhythm datavisualizer-panel card-panel">
       	<div class="euiText">
-          <h2 class="kuiSubTitle kuiVerticalRhythm">Metrics</h2>
+          <h2
+            class="kuiSubTitle kuiVerticalRhythm"
+            i18n-id="xpack.ml.datavisualizer.metricsTitle"
+            i18n-default-message="Metrics"
+          ></h2>
         </div>
 
         <div class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--justifyContentSpaceBetween euiFlexGroup--responsive euiFlexGroup--wrap">
           <div class="field-sort-controls euiFlexItem euiFlexItem--flexGrowZero">
             <div class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--responsive">
               <div ng-if="showAllMetrics === false" class="field-count-label euiFlexItem euiFlexItem--flexGrowZero">
-                <span class="field-emphasis">{{ metricCards.length }}</span>
-                <ng-pluralize count="metricCards.length" when="{'1':'field exists', 'other':'fields exist'}">fields exist</ng-pluralize> in documents (
-                <span class="field-emphasis">{{ totalMetricFieldCount }}</span> in total)
+                <span
+                  i18n-id="xpack.ml.datavisualizer.showMetricsDescription"
+                  i18n-default-message="{wrappedMetricCardsCount} {metricCardsCount, plural, one {field exists} other {fields exist}} in documents ({totalMetricFieldCount} in total)"
+                  i18n-values="{
+                      wrappedMetricCardsCount: '<span class=\'field-emphasis\'>' + metricCards.length + '</span>',
+                      metricCardsCount: metricCards.length,
+                      totalMetricFieldCount: '<span class=\'field-emphasis\'>' + totalMetricFieldCount + '</span>'
+                  }"
+                ></span>
               </div>
 
               <div ng-if="showAllMetrics === true" class="field-count-label euiFlexItem euiFlexItem--flexGrowZero">
-                <span class="field-emphasis">{{ metricCards.length }}</span>
-                <ng-pluralize count="metricCards.length" when="{'1':'field', 'other':'fields'}">fields</ng-pluralize>
-                (
-                <span class="field-emphasis">{{ populatedMetricFieldCount }}</span>
-                <ng-pluralize count="populatedMetricFieldCount" when="{'1':'exists', 'other':'exist'}">exist</ng-pluralize> in documents)
+                <span
+                  i18n-id="xpack.ml.datavisualizer.showAllMetricsDescription"
+                  i18n-default-message="{wrappedMetricCardsCount} {metricCardsCount, plural, one {field} other {fields}} ({wrappedPopulatedMetricFieldCount} {populatedMetricFieldCount, plural, one {exists} other {exist}} in documents)"
+                  i18n-values="{
```

Comment:
And again - the `<span>` tag should not be inside the `i18n-values` value

Reply:
With the last **master** <code>\<span\></code>  tag render correctly
![image](https://user-images.githubusercontent.com/25555662/50217901-32e20c00-039b-11e9-95bc-64062b18f9b8.png)